src/main/java/org/commcare/formplayer/application/UtilController.java

```diff
@@ -35,12 +35,20 @@
  */
 @RestController
 @EnableAutoConfiguration
-public class UtilController extends AbstractBaseController {
```

Comment:
I realized that removing this base class also removes the exception handling. Solved by moving the exception handling to a global advice: https://github.com/dimagi/formplayer/pull/1056